services.py

```
@@ -6,387 +6,313 @@
 
         if result.returncode == 0:
             return True
-        else:
-            console.print(f"[red]❌ Command failed[/red]")
-            if result.stderr:
-                console.print("[red]Error output:[/red]")
-                for line in result.stderr.splitlines():
-                    console.print(f"  [dim]{line}[/dim]")
-            return False
+
+        console.print(f"[red]❌ Command failed[/red]")
```

Ruff (0.14.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)